Create how-to-run-a-Design-System-team-ceremony-or-workshop (#30)
alphagov/design-system-team-docs: source/how-we-work/how-to-run-a-design-system-team-ceremony-or-workshop.html.md

---
title: How to run a Design System team ceremony or workshop
weight: 5
---

# How to run a Design System team ceremony or workshop 

This document details the principles, tools and resources we use when planning and running a ceremony or workshop on the Design System team.
## Principles

> There are some principles that are important to remember when planning and facilitating a ceremony or workshop on the Design System team;

Reviewer: ```suggestion
There are some principles that are important to remember when planning and facilitating a ceremony or workshop on the Design System team:
```

> - consider if there are asynchronous methods you can use to get the information you need. With remote working it is easy to experience video call fatigue.

Reviewer: For an iteration, but might be worth adding some docs on async communication?

Author: Yep agree

- share an agenda and session details ahead of time (including links to slide decks and collaboration tools). This helps participants prepare for the session and frame their thoughts - it can be difficult to think on the spot.
- consider the accessibility of the tools used in the session - we prefer to use Trello or Padlet.

> - make use of the ‘optional’ invite for those who don’t need to be there

Reviewer: ```suggestion
- make use of the ‘optional’ invite for those who don’t need to be there.
```

> - if it’s an open workshop, consider adding the meeting to the ‘Design System team events’ calendar which can be seen by all.

Reviewer: ```suggestion
- if it’s an open workshop, consider adding the meeting to the Design System team events calendar which can be seen by all. Ask a delivery manager for a link to the calendar.
```

Reviewer: Don't think we can share the calendar openly, even though it's access-controlled, so adding this as a workaround.

> - avoid booking over the lunch period (12-2pm). We generally avoid that period for meetings to give everyone an opportunity to have lunch / get away from their desks.

Reviewer: ```suggestion
- avoid booking over the lunch period (12-2pm). We generally avoid that period for meetings to give everyone an opportunity to have lunch and get away from their desks.
```

- things always take longer than you think they will! Scope your session carefully and ensure it ends on time.

## Ceremonies

> All team ceremonies live in the Design System team events calendar. This is to ensure multiple people can access the calendar entries to make amendments if the facilitator / owner is absent.

Reviewer: ```suggestion
All team ceremonies live in the Design System team events calendar. This is to ensure multiple people can access the calendar entries to make amendments if the facilitator or owner is absent.
```

Each squad has its own stand up, sprint planning, mid-sprint review and retro. The only combined ceremony is show & tell.

### Stand up

The team currently hold separate stand ups for each squad on the team. We have a different facilitator at each Google Meet stand up.

#### Design System squad

The Design System squad has a Google Meet stand up each day at 10am except on no-meeting days. It is up to the facilitator to decide the format of the stand up. From August 2023 the squad are trialling using the GitHub sprint board at every stand up.

There are a couple of formats that can be used:
- looking at the “Needs review” column only and identifying who is needed to review something and when they can do it
- reviewing the “Blocked” column and asking the team what’s needed to unblock something
- running through the “In progress” column and asking for updates
- reviewing the “Sprint backlog” and asking if anything can move into “In progress”
- focusing on just one epic and reviewing all cards in all columns related to that epic 

#### Frontend squad

> The Design System squad has a Google Meet stand up each day at 10am except on no-meeting days. It is up to the facilitator to decide the format of the stand up.

Reviewer: ```suggestion
The Frontend squad has a Google Meet stand up each day at 10am except on no-meeting days. It is up to the facilitator to decide the format of the stand up.
```

#### Keeping stand up focused

Do your best to ensure stand up keeps to 15 mins but this can be tricky when conversation starts flowing! Encourage the team to take detailed conversations offline.

### Show & tell

Show and tell occurs every other Tuesday. The facilitator will take a copy of the previous slide deck and prepare it ready for sharing. The team is prompted on the no-meeting Friday before to prepare slides. 

> We expect to hear from every epic in play at each show & tell.

Reviewer: ```suggestion
We expect to hear from every epic in play at each show & tell. Show & tells help us to:

- document projects, key decisions and activities
- communicate across the team
- share knowledge, understanding and ideas
- stop covering old ground in meetings
- get feedback on hypotheses, scrappy prototypes or conundrums

```

There are multiple slots available with 5 mins presenting time and 2 mins for questions (7 mins in total). 

When facilitating show & tell keeping time is probably the most important thing as it’s very easy to run over. Communicating clearly at the beginning of the session that each talk has 15 mins and you will intervene if it runs over helps with timekeeping.

> Introduce each of the speakers and their topics, and encourage questions afterwards.

Reviewer: ```suggestion
Start the recording. Introduce each of the speakers and their topics, and encourage questions afterwards.
```

> At the end of the session, share the slide deck.

Reviewer: ```suggestion
At the end of the session, share the slide deck and file the recording in the Recordings folder in our team drive.
```

Where a piece of work can help someone meet their professional development objectives, highlight that they might want to work on it – or assign it to them and let others on the team know they’ll be tackling it.

Note: This ceremony is fairly organic and has suited the team OK so far, but we have identified a need to increase collaboration and pairing on the team. So it should probably be amended / adapted as we tweak our processes to meet the needs of the team.

### Mid-sprint review

A mid-sprint review is a checkpoint halfway through a sprint. The session usually lasts 30 mins and checks progress against sprint goals, providing the opportunity to pivot if needed..

In a mid-sprint review, the squad asks the following questions of the sprint goals:
- have any risks or dependencies emerged?
- how can blockers be resolved?
- has the scope changed?
- has it been deprioritised? If so, why?
- has a new goal emerged?

Update the sprint board as a result of the discussion.

### Retrospectives

> Retros occur every other Tuesday for each squad. The purpose of retro is to reflect on the previous two weeks and identify ways the team can improve going forward. Therefore it’s useful if the whole team attends so we collectively agree to actions.

Reviewer: ```suggestion
Retros occur every other Tuesday for each squad. The purpose of retro is to reflect on the previous two weeks and identify ways the team can improve its processes going forward. Therefore it’s useful if the whole team attends so we collectively agree to actions.
```

Reviewer: This suggestion might be doing a lot of heavy-lifting, but the intent is to highlight how retros are more about our methods and processes. Feel free to remove it or reword it though!
